ci-operator/step-registry/ipi/conf/aws/oidc-creds-provision/ipi-conf-aws-oidc-creds-provision-commands.sh

```diff
@@ -10,16 +10,19 @@ infra_name=${NAMESPACE}-${JOB_NAME_HASH}
 export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
 REGION="${LEASED_RESOURCE}"
 
-# extract aws credentials requests from the release image
+echo "> Extract aws credentials requests from the release image"
 oc registry login
 oc adm release extract --credentials-requests --cloud=aws --to="/tmp/credrequests" "$RELEASE_IMAGE_LATEST"
 
-# create required credentials infrastructure and installer manifests
+echo "> Generated the following CredentialsRequests"
+ls /tmp/credrequests
+
+echo "> Create required credentials infrastructure and installer manifests"
 ccoctl aws create-all --name="${infra_name}" --region="${REGION}" --credentials-requests-dir="/tmp/credrequests" --output-dir="/tmp"
 
-# copy generated service account signing from ccoctl target directory into shared directory
-cp "/tmp/tls/bound-service-account-signing-key.key" "${TPREFIX}_bound-service-account-signing-key.key"
+echo "> Copy generated service account signing from ccoctl target directory into shared directory"
+cp -v "/tmp/tls/bound-service-account-signing-key.key" "${TPREFIX}_bound-service-account-signing-key.key"
```

Comment:
@abutcher should we need to fix here too?
```suggestion
cp -v "/tmp/tls/bound-service-account-signing-key.key" "${TPREFIX}/bound-service-account-signing-key.key"
```

Reply:
You can ignore, as you commented the TLS key should be in the root of install dir

Reply:
I think this one is correct as is since the key goes into the install directory as opposed to the manifests directory.